sdk/dns/Azure.ResourceManager.Dns/tests/DnsManagementClientBase.cs

```diff
@@ -6,12 +6,13 @@
 using Azure.Core.TestFramework;
 using Azure.Management.Resources;
 using Azure.ResourceManager.TestFramework;
```

Comment:
m-nash:
Can we remove any Dns test specific things to focus just on the modifications to the test base classes that all RPs will use

Reply:
@m-nash 
Yes, can do this. I just thought it be helpful to see one use case as well. 
If we find it easier to see the use case separately, I can do this